### ISSUE TITLE
Idea: [mini]kube Support

### ISSUE THREAD

**Issue by binarypie** [dart-lang/webdev#573]
This might be out of scope, but would webdev like [mini]kube support? Specifically to run the server inside a local cluster using the mount option to preserve hot reload and the like.  We run everything in minikube as our dev environment to make it closer to what production looks like. We have scripts that combined with #564 make this possible but we wouldn't mind pushing the functionality up if it sounds interesting. 

**High level concept:**
`webdev serve-k8`

- Would create dart webdev deployment
- Run webdev serve (with optional tls flags)
- Mount local directory for hot reload
- Expose port and optionally open web browser

**Comment by jakemac53** [comment]
I think this is probably a bit out of scope for this package - I think features which enable use cases such as this would be in scope though. Are you having to run a forked version of webdev in order to do this today or can you successfully wrap it in other tools?

**Comment by binarypie** [comment]
Other than the TLS support we can do this manually with some scripts wrapping a few kubectl commands. Once #565 is merged into mainline we'll deprecate our fork until we run into something that needs our attention.

**Comment by jakemac53** [comment]
Sounds good, going to close as not planned but do file issues if you hit any blockers. I think this would be a good candidate for a separate package potentially as well, if you wanted to publish/own such a thing :).
